frontend/smart-kickers-game/src/apis/goalManipulation.js

```diff
@@ -1,8 +1,17 @@
 import axios from 'axios';
 import config from '../config';
 
-export function pointsManipulation(teamID, action) {
-  axios.post(`${config.apiBaseUrl}/goal?action=${action}&team=${teamID}`).catch((err) => {
-    alert(err);
-  });
+export async function pointsManipulation(teamID, action) {
+  try {
+    let result = await axios.post(`${config.apiBaseUrl}/goal?action=${action}&team=${teamID}`);
+
+    return {
+      status: result.status,
+    };
+  } catch (e) {
+    return {
+      error: e,
+      status: e.response.status,
+    };
+  }
```

Comment:
Please handle the return values in the places when you use that function.

Reply:
Should it look like this `    
return {
      status: result.status,
    };
  } catch (e) {
    alert(e);
    return {
      error: e,
      status: e.response.status,
    };
  }`

Reply:
Not really, that function returns an object of some properties and they're not used anywhere. You use that function with the `onClick` event in the `GameResults` component, so these properties should be handled there.